src/skins/o2-new.tsx

```diff
@@ -0,0 +1,343 @@
+import {O2_NEW_SKIN} from './constants';
+import {applyAlpha} from '../utils/color';
+
+import type {GetKnownSkin, KnownSkin} from './types';
+
+export const palette = {
+    beyondBlue: '#0050FF',
+    beyondBlue90: '#001033',
+    beyondBlue70: '#0038B2',
+    beyondBlue45: '#0A73EB',
+    beyondBlue40: '#4C84FF',
+    beyondBlue30: '#80A7FF',
+    beyondBlue15: '#CCDCFF',
+    beyondBlue10: '#E5EDFF',
+    darkBlue: '#00008C',
+    o2BlueLight: '#82DCFA',
+    o2BlueLight30: '#C0EEFD',
+    o2BlueLight35: '#B4EAFC',
+    o2Green: '#00DC7D',
+    o2Green10: '#E5FBF2',
+    o2Green40: '#4CE7A4',
+    o2Green80: '#006338',
+    o2Yellow: '#FADC00',
+    o2Orange: '#FFA55A',
+    o2Orange10: '#FFF6EE',
+    o2Orange40: '#FFC08B',
+    o2Orange75: '#996336',
+    o2Pink: '#FA96FF',
+    o2Pink15: '#FEEAFF',
+    o2Pink30: '#FDCAFF',
+    o2Pink80: '#704373',
+    o2Red: '#FF5A5A',
+    o2Red10: '#FFEEEE',
+    o2Red20: '#FFCDCD',
+    o2Red40: '#FF8B8B',
+    o2Red55: 'EB0000',
+    o2Red65: '#BF4343',
+    grey20: '#F3F3F5',
+    grey30: '#D9D9DD', // nuevo para skeletons y otros
```

Comment:
what are these comments in Spanish?

Reply:
This file will be "reset" when we run the script to get skin from mistica-design json files. I can remove this comment but the file will be totally changed

Reply:
Don't forget to run the script before merging :)